AutomatedTesting/Gem/PythonTests/AWS/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -1,289 +1,289 @@
+
+
+def verify_operational_metrics(aws_metrics_utils: pytest.fixture,
+                               resource_mappings: pytest.fixture, start_time: datetime) -> None:
+    """
+    Verify that operational health metrics are delivered to CloudWatch.
+    :param aws_metrics_utils: aws_metrics_utils fixture.
+    :param resource_mappings: resource_mappings fixture.
+    :param start_time: Time when the game launcher starts.
+    """
+    aws_metrics_utils.verify_cloud_watch_delivery(
+        'AWS/Lambda',
+        'Invocations',
+        [{'Name': 'FunctionName',
+          'Value': resource_mappings.get_resource_name_id('AWSMetrics.AnalyticsProcessingLambdaName')}],
+        start_time)
+    logger.info('AnalyticsProcessingLambda metrics are sent to CloudWatch.')
+
+    aws_metrics_utils.verify_cloud_watch_delivery(
+        'AWS/Lambda',
+        'Invocations',
+        [{'Name': 'FunctionName',
+          'Value': resource_mappings.get_resource_name_id('AWSMetrics.EventProcessingLambdaName')}],
+        start_time)
+    logger.info('EventsProcessingLambda metrics are sent to CloudWatch.')
+
+
+def update_kinesis_analytics_application_status(aws_metrics_utils: pytest.fixture,
+                                                resource_mappings: pytest.fixture, start_application: bool) -> None:
+    """
+    Update the Kinesis analytics application to start or stop it.
+    :param aws_metrics_utils: aws_metrics_utils fixture.
+    :param resource_mappings: resource_mappings fixture.
+    :param start_application: whether to start or stop the application.
+    """
+    if start_application:
+        aws_metrics_utils.start_kinesis_data_analytics_application(
+            resource_mappings.get_resource_name_id('AWSMetrics.AnalyticsApplicationName'))
+    else:
+        aws_metrics_utils.stop_kinesis_data_analytics_application(
+            resource_mappings.get_resource_name_id('AWSMetrics.AnalyticsApplicationName'))
+
+@pytest.mark.SUITE_awsi
+@pytest.mark.usefixtures('automatic_process_killer')
+@pytest.mark.usefixtures('aws_credentials')
+@pytest.mark.usefixtures('resource_mappings')
+@pytest.mark.parametrize('assume_role_arn', [constants.ASSUME_ROLE_ARN])
+@pytest.mark.parametrize('feature_name', [AWS_METRICS_FEATURE_NAME])
+@pytest.mark.parametrize('profile_name', ['AWSAutomationTest'])
+@pytest.mark.parametrize('project', ['AutomatedTesting'])
+@pytest.mark.parametrize('region_name', [constants.AWS_REGION])
+@pytest.mark.parametrize('resource_mappings_filename', [constants.AWS_RESOURCE_MAPPING_FILE_NAME])
+@pytest.mark.parametrize('session_name', [constants.SESSION_NAME])
+@pytest.mark.parametrize('stacks', [[f'{constants.AWS_PROJECT_NAME}-{AWS_METRICS_FEATURE_NAME}-{constants.AWS_REGION}']])
+class TestAWSMetricsWindows(object):
+    """
+    Test class to verify the real-time and batch analytics for metrics.
+    """
+    @pytest.mark.parametrize('level', ['levels/aws/metrics/metrics.spawnable'])
```

Comment:
Can you speak to this name change? I don't see a metrics.spawnable like file in this PR

Reply:
The .spawnable file is the product assets of the prefab file and locates in the cache. The level is not loaded from source assets.